tests/manage/pv_services/space_reclaim/test_auto_reclaim_space_cronjob.py

```diff
@@ -220,9 +255,12 @@ def wait_for_cronjobs(self, cronjobs_exist, msg, timeout=60):
                 cmd="get reclaimspacecronjob",
                 namespace=self.namespace,
             ):
-                if (len(sample) > 1 and cronjobs_exist) or (
-                    len(sample) == 1 and not cronjobs_exist
-                ):
+                # if (len(sample) > 1 and cronjobs_exist) or (
+                #     len(sample) == 1 and not cronjobs_exist
+                # ):
```

Comment:
we can delete this commented line if they are not needed.

Reply:
Fixed. Thank you!